app/kumactl/cmd/uninstall/uninstall_ebpf.go

```diff
@@ -0,0 +1,232 @@
+package uninstall
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/kumahq/kuma/app/kumactl/pkg/client/k8s"
+	kumactl_cmd "github.com/kumahq/kuma/app/kumactl/pkg/cmd"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+
+	kuma_version "github.com/kumahq/kuma/pkg/version"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	typedbatchv1 "k8s.io/client-go/kubernetes/typed/batch/v1"
+	typedcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const (
+	AppLabel          = "kuma.io/bpf-cleanup"
+	BpfCleanupJobName = "kuma-bpf-cleanup"
+	BpfCleanupImage   = "kumahq/kuma-init"
```

Comment:
You need to make this configurable as often people will rename image or move them in different registries